examples/zxc.rs

```diff
@@ -7,7 +7,9 @@ use bellman::groth16::{
 use bellman::Circuit;
 use bls12_381::{Bls12, Scalar};
 */
+#[cfg(all(feature = "smt", feature = "zok"))]
```

Comment:
I don't actually know: are these still necessary if the example is gated on smt and zok in Cargo.toml?